templates/default-layout.lucius

```diff
@@ -1,30 +1,76 @@
+/** ************************************************************************* *
+ ** Pursuit CSS
+ **
+ ** This CSS file is structured into several sections, from general to
+ ** specific, and (mostly) alphabetically within the sections.
+ **
+ ** Several global element styles are used. This is not encouraged and should
+ ** be kept to a minimum. If you want to add new styles you'll most likely
+ ** want to add a new CSS component. See the Components section for examples.
+ **
+ ** CSS components use three simple naming ideas from the BEM system:
+ **   - Block:    `.my-component`
+ **   - Element:  `.my-component__item`
+ **   - Modifier: `.my-component.my-component--highlighted`
+ **
+ **   Example:
+ **   <div .my-component>
+ **     <div .my-component__item>
+ **     <div .my-component__item>
+ **   ...
+ **   <div .my-component.my-component--highlighted>
+ **     <div .my-component__item>
+ **     <div .my-component__item>
+ **
+ ** Components can be nested.
+ ** ************************************************************************* */
+
+
+/* Section: Document Styles
+ * ========================================================================== */
+
 html {
-  overflow-y: scroll;
```

Comment:
I'd quite like to keep this too, as without it everything will shift slightly to the side when moving from a page which isn't large enough to generate a scrollbar to one that is.